src/components/notifications/ui-notification-toast.tsx

```diff
@@ -74,13 +74,13 @@ export const UiNotificationToast: React.FC<UiNotificationProps> = ({
 
   const buttonsDom = useMemo(
     () =>
-      buttons?.map((button) => {
+      buttons?.map((button, buttonIndex) => {
         const buttonClick = () => {
           button.onClick()
           dismissThisNotification()
         }
         return (
-          <Button key={button.label} size={'sm'} onClick={buttonClick} variant={'link'}>
+          <Button key={buttonIndex} size={'sm'} onClick={buttonClick} variant={'link'}>
```

Comment:
Maybe a prefix would be usefull here? Something like `notification-${buttonIndex}`

Reply:
Why? Keys only need to be unique within their parent.